internal/pkg/service/common/dependencies/mocked.go

```diff
@@ -56,11 +56,12 @@ type mocked struct {
 
 type MockedConfig struct {
 	enableEtcdClient       bool
-	enableTasks            bool
-	enableDistribution     bool
 	enableDistributedLocks bool
 	enableOrchestrator     bool
 
+	tasksNodeID        string
+	distributionNodeID string
+
 	ctx         context.Context
 	clock       clock.Clock
```

Comment:
In the first commit is fix of mocked dependencies, ... the `nodeID` is always provided by a method argument, ... so the ID is always visible in tests/dependencies, there is no default value.